src/acl/connect_to_target_host.hpp

```diff
@@ -45,7 +45,7 @@
 #include "acl/module_manager/enums.hpp"
 #include "core/back_event_t.hpp"
 
-static inline unique_fd connect_to_target_host(Inifile & ini, const SessionReactor & session_reactor, ReportMessageApi& report_message, trkeys::TrKey const& authentification_fail)
+static inline unique_fd connect_to_target_host(Inifile & ini, const SessionReactor & session_reactor, ReportMessageApi& report_message, trkeys::TrKey const& authentification_fail, char *localIPAddress = nullptr)
```

Comment:
may I suggest `const char *`for `localIPAddress` ?

Reply:
"localIPAddress" will be set in the code so it cannot be const